gvm/protocols/gmpv7/gmpv7.py

```diff
@@ -5913,8 +5913,11 @@ def modify_task(
                     arg_type='list',
                 )
 
-            for alert in alert_ids:
-                cmd.add_element("alert", attrs={"id": str(alert)})
+            if len(alert_ids) == 0:
```

Comment:
```suggestion
            if not alert_ids:
```

https://stackoverflow.com/questions/53513/how-do-i-check-if-a-list-is-empty

Reply:
Doesn't this check if alert_ids is None?

Reply:
It also checks if it is empty ...
Maybe you can just do something like this (this would save one condition ...):
```
        if alert_ids:
            if not _is_list_like(alert_ids):
                raise InvalidArgumentType(
                    function=self.modify_task.__name__,
                    argument='alert_ids',
                    arg_type='list',
                )

            for alert in alert_ids:
                cmd.add_element("alert", attrs={"id": str(alert)})
        else:
            cmd.add_element("alert", attrs={"id": "0"})
```
What do you think @bjoernricks

Reply:
Yes it does. The if clause would be True for `alert_ids = None, alert_ids = "", alert_ids = []`